tests/javascript/Tiltfile

```diff
@@ -0,0 +1,59 @@
+# Some common test configurations to easily get up and running with Tilt for Tests
+# (see docs: https://docs.tilt.dev/tests_in_tilt.html)
+
+
+def _test_js(name, pkmanager, dir, deps=None, with_install=False, project_root='', ignore=None, extra_args=None):
+    if not project_root:
+        project_root = dir
+
+    install_deps = deps_for_package_manager(pkmanager, project_root)
+    resource_deps = []
+    if with_install:
+        install_resource = pkmanager + '-install'
+        install_cmd = 'cd {dir} && {pkmanager} install'.format(
+            dir=dir, pkmanager=pkmanager)
+        local_resource(install_resource, install_cmd, install_deps, allow_parallel=True)
```

Comment:
😬 😬  i do NOT, good catch.

Reply:
> or it'd probably be cheap and useful to change the build controller so that if A depends on B, A will not start building if B is pending / in progress, but that might be too much of a distraction for this

ugh yes every time i remember that this isn't in-built behavior i cry a little